scripts/emoji/translate.ts

```
@@ -0,0 +1,336 @@
+  console.log(chalk.white(`Total keywords: ${keywords.length}`));
+  console.log(chalk.blue(`Found in cache: ${cachedKeywords.length}`));
+  console.log(chalk.yellow(`Previously failed (skip): ${skipKeywords.length}`));
+  console.log(chalk.magenta(`Need translation: ${needTranslation.length}`));
+  console.log(chalk.green(`Direct translations: ${directKeywords.length}`));
+
+  // 캐시된 번역 처리
+  cachedKeywords.forEach((keyword: string) => {
+    const cached = cache[keyword];
+    if (cached && cached.ko && cached.ko !== keyword) {
+      dictionary[keyword] = cached;
+      console.log(chalk.blue(`[CACHE] ✓ ${keyword} -> ${cached.ko}`));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)